### DISCUSSION TITLE
8-0 Part 2 - polar projections broken

### DISCUSSION THREAD

**Discussion by travisz09** [ua-gist604b-f24/syllabus#45] [General]
For anybody working on assignment 8-0, be careful when working with the polar projections. While gdal will not throw any errors, and the output files appear correct, if you open the generated .tif files of the polar projections, all you will see is the transparent background layer. The map is not actually being projected correctly, and the image is empty.

After lots of testing and searching, I traced the source of the error back to this line.

`gdalwarp -t_srs EPSG:3976 -ts 7200 0 -r near -dstalpha -wo SOURCE_EXTRA=1000 -co COMPRESS=LZW NE1_50M_SR_W_SH60.vrt NE1_50M_SR_W_sh60_polarstereo.tif`

The Sea Ice Polar Stereographic South (EPSG:3976) projection seems to be the problem. IDK if this has been depreciated, or if it is just a bug, but I found that the Australian Antarctic Polar Stereographic (EPSG:3032) works well.

**Comment by kev-bone** [comment]
Agreed, 3032 seems good for south. Sea Ice Polar Stereographic North (EPSG: 3413) does work for the north pole though, so yeah don't know what is wrong with 3976.

https://nsidc.org/data/user-resources/help-center/guide-nsidcs-polar-stereographic-projection

**Reply by aaryno** [reply]
Tweak the extents (i.e., try 89.9, etc) to get it to work with a small sliver missing but using a different projection that works is good as well.

**Comment by khartman-nj** [comment]
$ gdalwarp -t_srs EPSG:3413 -ts 7200 0 -r near -dstalpha -wo SOURCE_EXTRA=1000 -co COMPRESS=LZW NE1_50M_SR_W_SH60.vrt NE1_50M_SR_
W_sh60_polarstereo.tif
I ran this after creating the new vrt file but I cant seem to get the north project any insights.

**Reply by vmwielgus** [reply]
Did you use the correct coordinates when you created your VRT for the north pole? You might be getting an error because you're still referencing the SH60.vrt in your code above. I'd say try creating the north pole VRT again but name it NH60.vrt this time. Then in the line of code above, change SH60.vrt to NH60.vrt and change the last part of the file name to nh60_polarstereo.tif to get a new tif for the north.
